core/chaos-workers/chaos-worker/src/main/kotlin/io/zeebe/chaos/chaosMain.kt

```diff
@@ -55,13 +55,16 @@ fun main() {
     scriptPath.listFiles { file -> file.extension == SHELL_EXTENSION }!!
         .map { it.name }
         .filterNot { it.contains("utils") }
+        .filterNot { it.equals(AwaitMessageCorrelationHandler.JOB_TYPE) }
         .filterNot { it.equals(AwaitProcessWithResultHandler.JOB_TYPE) }
         .filterNot { it.equals(DeployMultipleVersionsHandler.JOB_TYPE) }
         .forEach { script ->
             LOG.info("Start worker with type `$script`")
             zeebeClient.newWorker().jobType(script).handler(::handler).open()
         }
 
+    zeebeClient.newWorker().jobType(AwaitMessageCorrelationHandler.JOB_TYPE)
+        .handler(AwaitProcessWithResultHandler()).open()
```

Comment:
I assume you want to call the other one :) 

```suggestion
        .handler(AwaitMessageCorrelationHandler()).open()
```